<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExchangeSinglePairCellWidget</class>
 <widget class="QWidget" name="ExchangeSinglePairCellWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>110</width>
    <height>30</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="iconLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="pairLabel">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>5</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="checkLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>11</y>
     <width>11</width>
     <height>8</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ui/wallet_ui/checkMark.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
